modules/perception/camera/lib/obstacle/detector/yolo/region_output.cu

```diff
@@ -388,13 +388,19 @@ const float *get_gpu_data(bool flag, const base::Blob<float> &blob) {
   return flag ? blob.gpu_data() : nullptr;
 }
 
-void get_objects_gpu(const YoloBlobs &yolo_blobs, const cudaStream_t &stream,
-                     const std::vector<base::ObjectSubType> &types,
-                     const NMSParam &nms, const yolo::ModelParam &model_param,
-                     float light_vis_conf_threshold,
-                     float light_swt_conf_threshold,
-                     base::Blob<bool> *overlapped, base::Blob<int> *idx_sm,
-                     std::vector<base::ObjectPtr> *objects) {
+int get_objects_gpu(const YoloBlobs &yolo_blobs, const cudaStream_t &stream,
+  const std::vector<base::ObjectSubType> &types,
+  const NMSParam &nms, const yolo::ModelParam &model_param,
+  float light_vis_conf_threshold,
+  float light_swt_conf_threshold,
+  base::Blob<bool> *overlapped, base::Blob<int> *idx_sm,
+  const std::map<base::ObjectSubType, std::vector<int>> &indices_const,
+  const std::map<base::ObjectSubType, std::vector<float>> &conf_scores_const) {
```

Comment:
It would be better to indent 4 spaces for the function param lines, or keep the same style as the older one.